README.md

```diff
@@ -13,33 +13,24 @@ between both the server and the client) for SEO reasons.
 [![Dependency Status](https://david-dm.org/quran/quran.com-frontend.svg)](https://david-dm.org/quran/quran.com-frontend) [![devDependency Status](https://david-dm.org/quran/quran.com-frontend/dev-status.svg)](https://david-dm.org/quran/quran.com-frontend#info=devDependencies)
 [![Code Climate](https://codeclimate.com/github/quran/quran.com-frontend.png)](https://codeclimate.com/github/quran/quran.com-frontend)
 
-## Getting started
-Simply clone this repo, then run `npm install` to install all the required node_modules.
-From there, you are ready to go!
+## How to contribute
+We trust that
+you will not steal this, this is at the end of the day for the sake of Allah and we
```

Comment:
thanks for putting this in, perhaps renaming 'stealing' to 'copying the code'. Stealing might be strong (especially that Islamically it is a huge sin and we don't want to blame anyone or accuse anyone of such a sin).